mappings/net/minecraft/data/server/loot_table/DesertWellLootTableGenerator.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_uakojvce net/minecraft/data/server/loot_table/DesertWellLootTableGenerator
```

Comment:
I'm not sure for `DesertWellLootTableGenerator` (we have Desert Well Archaeology & DesertPyramid Archaeology). Maybe `DesertArchaeologyLootTableGenerator` instead.

```suggestion
CLASS net/minecraft/unmapped/C_uakojvce net/minecraft/data/server/loot_table/DesertArchaeologyLootTableGenerator
```

Reply:
sure!